src/Helpers/OrderBuilderTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace GingerPluginSdk\Helpers;
+
+use GingerPluginSdk\Entities\Order;
+use GingerPluginSdk\Response\GingerHTTPResponse;
+use GingerPluginSdk\Response\GingerHTTPResponseBody;
+use JetBrains\PhpStorm\ArrayShape;
+
+trait OrderBuilderTrait
+{
+    protected Order $order;
+
+    public function setOrder(Order $order): static
+    {
+        $this->order = $order;
+        return $this;
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function debugOrder()
+    {
+        if (!isset($this->order)) {
+            throw  new \Exception("Firstly execute the `setOrder` method, to debug the order");
+        }
+        print_r($this->order->toArray());
+        die(200);
+    }
+
+    /**
+     * @throws \Exception
+     */
+    #[ArrayShape(['status' => "bool", 'body' => "array"])] public function sendOrder(): array
+    {
+        if (!isset($this->order)) {
+            throw  new \Exception("Firstly execute the `setOrder` method, to save the order");
+        }
+        try {
+            $success_response = $this->api_client->createOrder($this->order->toArray());
+            $response = new GingerHTTPResponse(
+                status: true,
+                body: new GingerHTTPResponseBody(
+                    code: '201',
+                    data: $success_response
+                )
+            );
+        } catch (\Exception $exception) {
+            $trace = $exception->getTrace();
+            $error_body = current(current($trace)["args"])["error"];
+            $additional_error_data = [
+                'property_description' => $error_body['property_description'] ?? null,
+                'property_path' => $error_body['property_path'] ?? null
+            ];
+            $response = new GingerHTTPResponse(
```

Comment:
It seems strange to convert any exception to a HTTP response. Perhaps you can transform to your own exception in case of failure.